src/store/currency/currency-reducers.ts

```diff
@@ -0,0 +1,32 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import { createEntity } from '../create-entity';
+import { loadExchangeRates, loadTezosExchangeRate } from './currency-actions';
+import { currencyInitialState, ExchangeRatesState } from './currency-state';
+
+export const currencyReducers = createReducer<ExchangeRatesState>(currencyInitialState, builder => {
+  builder.addCase(loadExchangeRates.submit, state => ({
+    ...state,
+    tokensExchangeRates: createEntity({}, true)
```

Comment:
you are updating value on "`submit`" action, this will cause blinking of $ values everywhere